src/FeatureExtractor/Mobilenet.js

```diff
@@ -17,6 +16,7 @@ import { saveBlob } from '../utils/io';
 import callCallback from '../utils/callcallback';
 
 const IMAGE_SIZE = 224;
+const BASE_URL = 'https://storage.googleapis.com/tfjs-models/tfjs/mobilenet_v';
```

Comment:
if possible you should try to use TFHub, see the URLs here: https://github.com/tensorflow/tfjs-models/blob/master/mobilenet/src/index.ts#L31

We're going to deprecate the googleapis.com links at some point in the near future in favor of TF Hub :)

Reply:
oh, apologies, you're using this as a layers model. we only have the graph model on TF Hub. You can continue to use this for now, or alternatively you can find the activation in the graph model you load from TF Hub. Either is fine.

Reply:
@nsthorat - oh! excellent note. Thank you for pointing this out. Once we get ml5 up to tf1.0+ we can go through and make sure to switch over asap. Thanks so much!

Reply:
@nsthorat  - 

> oh, apologies, you're using this as a layers model. we only have the graph model on TF Hub. You can continue to use this for now, or alternatively you can find the activation in the graph model you load from TF Hub. Either is fine.

+ Noted! Thank you. Good to know there are options :)